src/network/internal_types.rs

```diff
@@ -28,9 +28,7 @@ pub struct TearDownNetwork {
 }
 
 #[derive(Clone, Debug)]
-pub struct PortForwardConfig {
-    //  network object
-    pub net: types::Network,
+pub struct PortForwardConfig<'a> {
```

Comment:
Why is `lifetime` added, do we need this or is this added as a precaution ?

Reply:
yes we need this, the linter will complain if this was added when it is not required.